explorer/types.go

```diff
@@ -398,8 +398,9 @@ type HostMetrics struct {
 	// equal to TotalStorage - RemainingStorage)
 	RemainingStorage uint64 `json:"remainingStorage"`
 
-	Settings   rhpv2.HostSettings   `json:"settings"`
-	PriceTable rhpv3.HostPriceTable `json:"priceTable"`
+	Settings      rhpv2.HostSettings   `json:"settings"`
+	PriceTable    rhpv3.HostPriceTable `json:"priceTable"`
+	RHPV4Settings rhpv4.HostSettings   `json:"hostSettings"`
```

Comment:
Do we want to `omitempty` on `settings` and `hostSettings`? I also considered started a discussion on the naming but this is probably fine, `renterd` uses `settings` and `v2settings`.

Reply:
Unless this is already exposed anywhere else as `hostSettings` I'd probably indeed go for `v2Settings`. Seems a bit more straightforward to me.

Also should be checked for in some test to make sure it is set.